hooks/emitter/authz.go

```diff
@@ -80,6 +81,7 @@ func (h *Hook) handleMsgExec(
 			h.handleMsgReportData(ctx, txHash, msg, grantee)
 		case *feedstypes.MsgSubmitSignalPrices:
 			h.handleFeedsMsgSubmitSignalPrices(ctx, txHash, msg, grantee)
+		case *tsstypes.MsgSubmitSignature, *tsstypes.MsgSubmitDEs:
```

Comment:
We still need to do a handler for these messages. Can you recheck it?
In the original code, it would fall back to default and do `handleMsg,` which we have the logic to handle `MsgSubmitSignature. '